Getting Started with Form (#1485)
DevExpress/devextreme-documentation: concepts/05 UI Components/Form/01 Getting Started with Form/40 Organize the Form Items/20 In Groups.md

> You can add groups to better organize the data entry form. To add a group, create a [group item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and nest other items within it, as shown in the code below. Group items can nest items of any type, including other group items. You can configure each group's layout separately. 

Reviewer: ```suggestion
You can use groups to organize the data entry form. To add a group, create a [group item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and nest other items in it as shown in the code below. Items of any type can be nested, including other group items. You can configure each group's layout separately. 
```

> Form items can be grouped and displayed under a single caption. To implement this layout, create a [group item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and nest other items within it, as shown in the code below. Group items can nest items of any type, including other group items. You can configure each group's layout separately. 

Reviewer 2: ```suggestion
You can add group boxes to better organize the data entry form. To add a group box, create a [group item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and nest other items within it, as shown in the code below. Group items can nest items of any type, including other group items. You can configure each group's layout separately. 
```

Author: I had to google what 'group box' is, and it seems like this name(type?) is a term from the world of desktop development. Maybe we just remove one confusing word? 

>You can add groups to better organize the data entry form. To add a group, ...

> The following code creates two groups, each group occupies a separate column. The resulting layout looks as follows:

Reviewer 3: The word "group" is overused here.

> To divide items into groups, declare items with the "group" [itemType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#itemType) in the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array. Items of this type can contain [simple items](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/), other groups, [tabs](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/TabbedItem/tabs/) or [empty items](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/EmptyItem/). To put an item into a group, declare this item inside the group as shown in the example below. You can configure each group's layout separately. 
> 
> The following code creates two groups, each one occupies a separate column. The resulting layout looks as follows:

Reviewer 3: Now the word "items" is all over the place :) How about this?

```suggestion
Form items can be grouped and displayed under a single caption. To implement this layout, create a [group item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/) and nest other items within it, as shown in the code below. Group items can nest items of any type, including other group items. You can configure each group's layout separately. 

The following code creates two groups, each occupies a separate column. The resulting layout looks as follows:
```
